### PR TITLE
add nokov support

### PR DESCRIPTION
**Pull request by 15871812598**
support nokov Motion capture attitude and position。

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,6 +18,9 @@ def package_files(directory):
                 'modules/mavproxy_magical/data/*.obj',
                 'modules/mavproxy_fieldcheck/*.txt',
                 'tools/graphs/*.xml',
+                'modules/mavproxy_nokov/nokov/*',
+                'modules/mavproxy_nokov/nokov/lib/*',
```
Comment (tridge) [inline setup.py:22]: why add in these subdirectories? they don't seem to exist

**Review by tridge — CHANGES_REQUESTED** [review]

File: MAVProxy/modules/mavproxy_nokov/__init__.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
+"""
+use nokov data to provide motion capture data
+it works with nokov software
+"""
+
+import time
+
+from MAVProxy.modules.lib import mp_module
+from MAVProxy.modules.lib import mp_settings
+from MAVProxy.modules.mavproxy_nokov.nokov import nokovsdk
```
Comment (tridge) [inline MAVProxy/modules/mavproxy_nokov/__init__.py:10]: where is nokovsdk?

**Comment by tridge** [comment]
nice!
would you be willing to do a blog post on discuss.ardupilot.org ? That would then provide a place for you to answer setup questions for other users in replies on the blog post

File: setup.py (hunk 1 above)
Comment (15871812598) [reply setup.py:22]: Yes, we did not submit the nokovsdk.

File: MAVProxy/modules/mavproxy_nokov/__init__.py (hunk 2 above)
Comment (15871812598) [reply MAVProxy/modules/mavproxy_nokov/__init__.py:10]: We plan to send the nokovsdk directly to the users who need it.Is that ok?

File: setup.py (hunk 1 above)
Comment (15871812598) [reply setup.py:22]: The SDK has been committed.

File: MAVProxy/modules/mavproxy_nokov/__init__.py (hunk 2 above)
Comment (15871812598) [reply MAVProxy/modules/mavproxy_nokov/__init__.py:10]: The SDK has been committed.

**Comment by 15871812598** [comment]
#The document of nokov module [#4356](https://github.com/ArduPilot/ardupilot_wiki/pull/4356)

**Comment by 15871812598** [comment]
> 

I have submitted the document to wiki, but it has not been approved yet.

**Comment by 15871812598** [comment]
> nice! would you be willing to do a blog post on discuss.ardupilot.org ? That would then provide a place for you to answer setup questions for other users in replies on the blog post

I have submitted the document to wiki, but it has not been approved yet.

**Comment by 15871812598** [comment]
> nice! would you be willing to do a blog post on discuss.ardupilot.org ? That would then provide a place for you to answer setup questions for other users in replies on the blog post

I have submitted the document to ardupilot_wiki, but it has not been approved yet.

**Comment by tridge** [comment]
@15871812598 what is the licensing on the SDK? We don't normally bring binary libraries into MAVProxy, and it is likely a conflict with the GPLv3 license if source is not available.
Could you instead have a shell script or python script that imports the SDK? Or have a README.md in the mavproxy_nokov module directory that gives instructions on installing?

**Comment by 15871812598** [comment]
@tridge 
hello!
This is a SDK of C++,if can't bring binary libraries into MAVProxy,can I just remove the SDK.When the user needs it, we give SDK directly to him.

**Comment by tridge** [comment]
> This is a SDK of C++,if can't bring binary libraries into MAVProxy,can I just remove the SDK.When the user needs it, we give SDK directly to him.

yes, that is fine, and that would solve the licensing issue. Thank you!

**Comment by 15871812598** [comment]
Prepare to create one new PR.

**Pull request closed without merging**